src/sass/feature/_base.scss

```diff
@@ -4,50 +4,52 @@
 @use 'core' as *;
 
 .#{$prefix}-feature {
+  display: flex;
+  flex-direction: column;
+
+  &--horizontal {
+    flex-direction: row;
+
+    .#{$prefix}-feature__image {
+      margin-right: $spacing-sm;
+    }
```

Comment:
In the horizontal version we'll need to control the flex-shrink/grow properties here. The `__image` container shouldn't shrink as the outer container get's smaller. That will mean setting a `max-width` on the `__image` container and setting `flex-shrink: 0`. For now I think we can pick a somewhat arbitrary value for the `max-width` something around 160-180 rendered pixels will be good. So whatever that is in rem. I would make the width some multiple of a `$spacing-*` unit.

We want to avoid the `__image` container shrining like this on small screens:
![image](https://user-images.githubusercontent.com/5590034/75354265-b68a6900-587a-11ea-868d-6fd740e24795.png)

I would also think about the way top magins are handled in the `__body` here. The top of the content should be flush with the top of the `__image` container. (see screenshot ☝️ ). I'm not sure why the first `AUTHOR` tag is getting top margin there since you are using the `*+*` selector here on the `__body` element.